app/models/edition.rb

```diff
@@ -67,71 +65,14 @@ class Edition < ApplicationRecord
   validates_with StateForDocumentValidator
   validates_with RoutesAndRedirectsValidator
 
-  # Temporary code until we remove content_id and locale fields
-  before_save do
-    next unless document
-    self.content_id = document.content_id unless content_id == document.content_id
-    self.locale = document.locale unless locale == document.locale
-  end
-
-  after_save do
-    lock_version = LockVersion.find_or_create_by(target: self)
-    if document.stale_lock_version < lock_version.number
-      lock_version.update! number: document.stale_lock_version
-    end
-  end
-
-  # Temporary code until we kill Location, State, Translation, and
-  # UserFacingVersion
-  after_save do
-    if changes[:base_path] && changes[:base_path].last
-      Location.find_or_initialize_by(content_item_id: id)
-        .update!(base_path: changes[:base_path].last)
-    end
-
-    if changes[:base_path] && !changes[:base_path].last
-      Location.find_by(content_item_id: id).try(:destroy)
-    end
-
-    if changes[:state]
-      State.find_or_initialize_by(content_item_id: id)
-        .update!(name: changes[:state].last)
-    end
-
-    if changes[:locale]
-      Translation.find_or_initialize_by(content_item_id: id)
-        .update!(locale: changes[:locale].last)
-    end
-
-    if changes[:user_facing_version]
-      UserFacingVersion.find_or_initialize_by(content_item_id: id)
-        .update!(number: changes[:user_facing_version].last)
-    end
-  end
-
-  before_save { ensure_document }
+  def as_json(options = {})
```

Comment:
I think now we're removing the backwards compatible behaviour we should look at keeping this as just serializing edition as json and if we need different logic here we should have it coming through a presenter